roles/allocations/templates/allocations_balance_service.conf.j2

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+debug: False
+
+[serviceaccount]
+auth_url: {{ keystone_public_url }}/v3
+auth_users: {{ allocations_auth_users }}
+
+[redis]
+host: {{ redis_host }}
+port: 6379
```

Comment:
I think `{{ redis_port }}` might be already a built-in variable in Kolla-Ansible we could use.